js/resources.js

```diff
@@ -1,40 +1,28 @@
 $(document).ready(function(){
```

Comment:
Can you add some comments for this section? Perhaps API docs for the functions?

Also, does this file pass linting?